code/datums/components/jump.dm

```diff
@@ -98,11 +108,12 @@
 	var/original_pass_flags = jumper.pass_flags
 
 	SEND_SIGNAL(jumper, COMSIG_ELEMENT_JUMP_STARTED, effective_jump_height, effective_jump_duration)
-	jumper.adjustStaminaLoss(stamina_cost)
+	if(use_stamina)
+		jumper.adjustStaminaLoss(stamina_cost)
```

Comment:
can't we just do this in start_jump? There's no other scenario where we using stam.

Reply:
sure